### DISCUSSION TITLE
Client bundle compiled with errors therefore further build is impossible.

### DISCUSSION THREAD

**Discussion by namnguyenthanhwork** [facebook/docusaurus#9285] [Q&A]
## Prerequisites
- I'm using the latest version of Docusaurus.
- I have tried the `npm run clear` command.
- I have tried `rm -rf node_modules yarn.lock package-lock.json` and re-installing packages.
- I have read the console error message carefully (if applicable).
## Description
I just edit content in mdx file, change content html footer in docusaurus.config.js file
<img width="1372" alt="image" src="https://github.com/facebook/docusaurus/assets/78526193/03c59728-e71b-47f1-8b9a-9839871b9b54">
<img width="1384" alt="image" src="https://github.com/facebook/docusaurus/assets/78526193/ebfa0bc7-7af0-48c8-af1a-c8a712b1a631">

After that, i run command `npm run build` on local machine (no error)

<img width="579" alt="image" src="https://github.com/facebook/docusaurus/assets/78526193/73a5587b-d338-418e-aac3-a712b2517ec3">

But when i merge on github and run build production (using vercel), it show error

<img width="1197" alt="image" src="https://github.com/facebook/docusaurus/assets/78526193/89b49fa7-d461-4e7f-a760-39fc92258de5">

I don't know why vercel cannot build. Can you help me ? Thank you.

[Update]
I try copy source project to new repo and deploy in vercel (still error) but when i deploy in netlify (successfully)
<img width="1301" alt="image" src="https://github.com/facebook/docusaurus/assets/78526193/7bef30bb-d848-4fec-b30c-7776318c49e1">

**Comment by slorber** [comment]
I don't know, weird. Maybe it's related to Vercel Node.js version?

**Reply by namnguyenthanhwork** [reply]
I don't know, but when i check another mdx file, i remove link tel -> it's working correctly (deploy in vercel successfully).
Why markdown syntax `**📞 Điện thoại:** [(028) 38364748](tel:02838364748)` is bug when deployed in Vercel. I never have bug before although I still keep `**📞 Điện thoại:** [(028) 38364748](tel:02838364748)` before.

<img width="761" alt="image" src="https://github.com/facebook/docusaurus/assets/78526193/bc584255-35a7-40b1-8799-c814b21ab7f4">

**Reply by slorber** [reply]
I don't know. It's impossible to debug anything without knowing vercel environment, Docusaurus version and everything. If you can reproduce the problem locally using the CI env, then we'll investigate

**Reply by slorber** [reply]
Possibly the same bug as https://github.com/facebook/docusaurus/issues/8641 ?

**Reply by namnguyenthanhwork** [reply]
Yes, it is. In vercel just show `TypeError: Invalid URL`, don't show file or line bug, so i don't know fix them (local machine is nomal when i run command `npm run build`). 
I take many time to find file and line make a bug.
